traffic_ops/app/lib/API/DeliveryService/KeysUrlSig.pm

```diff
@@ -98,7 +118,7 @@ sub copy_url_sig_keys {
 	if ( defined($url_sig_key_values_json) ) { # verify we got keys copied
 		# Admins can always do this, otherwise verify the user
```

Comment:
this comment is not true. admins cannot always do this anymore. not if use_tenancy=1